scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -102,3 +102,5 @@ scripts.linters.pre_commit_linter_test
 scripts.pre_push_hook_test
 scripts.run_e2e_tests_test
 scripts.run_frontend_tests_test
+scripts.install_third_party_libs_test
+scripts.run_mypy_checks_test
```

Comment:
since, just after 4days of this PR merged, I will start working on the python scripts deprecation and I am very tight with my final deadlines, so @gp201 asked to directly ignore the branch coverage for now. the changes that are breaking the branch coverage are just if-else conditions for maintaining the transition between docker and python setup

Reply:
@Shivkant-Chauhan Please add comment saying that it will be addressed in the upcoming PR.